allennlp/commands/elmo.py

```diff
@@ -0,0 +1,203 @@
+"""
+The ``elmo`` subcommand allows you to make bulk ELMo predictions.
+
+Given a pre-processed input text file, this command outputs the internal
+layers used to compute ELMo representations to a single (potentially large) file.
+
+The input file is previously tokenized, whitespace separated text, one sentence per line.
+The output is a hdf5 file (http://docs.h5py.org/en/latest/) where each
```

Comment:
This URL needs to be formatted like this: https://github.com/allenai/allennlp/blob/master/allennlp/models/semantic_role_labeler.py#L25